src/modules/todo.js

```diff
@@ -0,0 +1,110 @@
+const form = document.querySelector('.form');
```

Comment:
- [ ] [optional] When modifying the task the changes don't update in the local storage, kindly try to fix the edit function